components/RandomQueue.js

```diff
@@ -19,22 +27,27 @@ const STUDENTS = [
   'Karlie Kloss',
   'Elon Musk',
   'Grace Hopper',
-  'Steve Jobs',
```

Comment:
rip 😢

Reply:
Yeah, one comment was that was a little freaky, sort of agree. :cry: